rust/src/interpreter/stack.rs

```diff
@@ -0,0 +1,149 @@
+use evmc_vm::{StatusCode, StepStatusCode};
+
+use crate::types::u256;
+
+#[derive(Debug)]
+pub struct Stack(Vec<u256>);
+
+impl Stack {
+    pub fn from(inner: Vec<u256>) -> Self {
```

Comment:
nit: call it newFrom, or newWith. Just for coherence with the rust style.

Reply:
Agreed.